### ISSUE TITLE
Tarea 3 backpack 

### ISSUE THREAD

**Issue by Ctrbcb** [IIC2133-PUC/2023-1#103]
Hola, tengo una duda respecto al objetivo de primera parte de la tarea, según entiendo, debemos minimizar la cantidad de agentes y maximizar el peso total entre todos los items que llevan los agentes?

**Comment by CarloGauss33** [comment]
Nope, Solo minimizar la cantidad de agentes. Debes repartir TODOS los items
